src/core/control/Control.cpp

```diff
@@ -1611,15 +1611,27 @@ void Control::setViewPairedPages(bool enabled) {
     scrollHandler->scrollToPage(getCurrentPageNo());
 }
 
+void Control::setViewFullscreenMode(bool enabled) {
+    if (enabled) {
+        this->loadViewMode(1);
```

Comment:
Please use declared constexpr instead of plain numbers. Those really are view mode id's and should be immediately understandable to any coder.

Reply:
Oh ,I thought I had replaced all those occurrences. I'll fix this one!